src/views/users/Users.jsx

```diff
@@ -413,28 +460,14 @@ class Users extends React.Component {
   componentDidMount() {
     this.props.getAllUsers();
   }
-  componentDidUpdate() {
-    //     const {
-    //         updateUserResponse,
-    //         deleteUserResponse
-    //     }
-    // } = this.props;
-    //     let  response = this.props.updateUserResponse;
-    //     if (response.data ) {
-    //         swal.fire(getAlertToast('success', response.data));
-    //     }
-    //const del_data = newProps.deleteUserResponse.data;
-    // const updateUserResponse = newProps.updateUserResponse;
-    // if (
-    //     updateUserResponse &&
-    //     updateUserResponse !== this.props.updateUserResponse
-    // ) {
-    //     let isError = updateUserResponse.error;
-    //     let update_data = updateUserResponse.data;
-    //     let text = isError ? update_data.error.message : update_data;
-    //     let type = isError ? 'error' : 'success';
-    //     swal.fire({ text: text, type: type });
-    // }
+  componentDidUpdate(newProps) {
+    const { updateUserResponse, deleteUserResponse } = this.props;
+    if (newProps.updateUserResponse !== updateUserResponse) {
+      swal.fire(getAlertToast('success', updateUserResponse.data));
+    }
+    if (newProps.deleteUserResponse !== deleteUserResponse) {
+      swal.fire(getAlertToast('success', deleteUserResponse.data));
+    }
```

Comment:
if api is successful you can fire toasts from actions itself, this logic is not recommended